### ISSUE TITLE
Post-commit hooks not working in some cases?

### ISSUE THREAD

**Issue by jrudolph** [buildhive/buildhive#8]
For example:

https://buildhive.cloudbees.com/view/My%20Repositories/job/jrudolph/job/sbt-cross-building/

I pushed to this repository several times but only the pull request seemed to have triggered a new build.

**Comment by kohsuke** [comment]
Can you check if https://buildhive.cloudbees.com/github-pull-request-hook/ and https://buildhive.cloudbees.com/github-webhook/ are in the post-receive hook URLs?

We only update this when you create a project. I'm afraid of constantly updating them as people might manually remove them? (Or is that even a valid concern, when they can just delete a job from BuildHive altogether?)

**Comment by jrudolph** [comment]
Yes, they were added and they are still in there. The funny thing is, that the pull request hook actually seems to work. Maybe I can try the other hook manually and see what it returns?

**Comment by jrudolph** [comment]
Still not working for any of my projects. I captured a post-receive request and replayed manually against the build-hive server which is returning "200 OK" but still no new build is triggered. Is there anything else I can do to check what is going on there?

Maybe there's some matching code in buildhives which isn't able to correctly match the incoming update message to the correct buildhive project?

I used this simple html formular to test the request against build-hive:

https://gist.github.com/2829222
